src/components/FormField.tsx

```diff
@@ -10,7 +10,7 @@ import {
 interface FormFieldProps {
   id: string;
   label: string;
-  type: string;
+  type?: string;
```

Comment:
Why is the type optional now?

Reply:
This was a mistake where the `type` prop wasn't used in this component before we started on this project. I didn't realize it until I was updating this. I made it optional here but in a later PR that was specific to the `FormField` component, the type defaults to `text`.
PR #87 https://github.com/NYPL/nypl-library-card-app/pull/87/files#diff-2c1d733102374d1cb59bc39ffbacd365R36